app/Gists/MarkdownRenderable.php

```diff
@@ -0,0 +1,17 @@
+<?php namespace Gistlog\Gists;
+
+use Michelf\MarkdownExtra;
+
+trait MarkdownRenderable {
+
+    public function render($html)
```

Comment:
I think this parameter should probably be `$markdown` and the local `$md` variable should be `$html` since we are converting *from* markdown *to* HTML ya?

Also what do you think of giving it a name like `renderFromMarkdown` or something? Just to help avoid collisions in the Gistlog class, because I'm thinking the "<pre><code>" crap I added earlier will probably get extracted to another method with a name using the word "render" in it as well so I wonder if we should be more explicit.